packages/icons-react/examples/storybook/.storybook/config.js

```diff
@@ -1,4 +1,10 @@
 import { configure } from '@storybook/react';
+import { setOptions } from '@storybook/addon-options';
+
+setOptions({
```

Comment:
I followed the precedent from `carbon-components-react`:
https://github.com/IBM/carbon-components-react/blob/master/.storybook/config.js#L8